lib/matplotlib/axes/_axes.py

```diff
@@ -2858,7 +2866,7 @@ def errorbar(self, x, y, yerr=None, xerr=None,
         else:
             xuplims = np.asarray(xuplims, bool)
 
-        everymask = np.arange(len(x)) % errorevery == 0
+        everymask = np.arange(len(x)) % errorevery == offset
```

Comment:
This is not quite what the documentation states above (although the what's new does say it more explicitly.) Consider `errorevery = 2`, `offset = 2`. Nothing in the above documentation states that it won't work, but `everymask` will be all `False` here.